ada/language/ast.py

```diff
@@ -11285,7 +11300,7 @@ def potential_dottable_type():
         """
         return Entity.params.at(0)._.type_expr._.element_type
 
-    @langkit_property(return_type=BaseTypeDecl.entity)
+    @langkit_property(return_type=BaseTypeDecl.entity, public=True)
```

Comment:
If that's necessary, it's probably better to write an accessor that does the work on the canonical declaration of the subprogram, like for `primitive_subp_types` and co.